src/main/java/org/zone/commands/structure/invite/ZoneInviteAcceptCommand.java

```diff
@@ -63,7 +62,7 @@
                     if (!(invites.contains(player.uniqueId()))) {
                         return;
                     }
-                    zone.getMembers().addMember(DefaultGroups.HOME_OWNER, player.uniqueId());
+                    zone.getMembers().addMember(DefaultGroups.MEMBER, player.uniqueId());
```

Comment:
What is member?

Reply:
So this should be joining the group with a GroupKey that is for the default group. We should never be creating a whole new group just for a special case